src/hdf5_time_series_storage.jl

```diff
@@ -293,54 +276,22 @@ function get_data_type(ts::TimeSeriesData)
 end
 
 function _write_time_series_attributes!(
-    storage::Hdf5TimeSeriesStorage,
-    ts::T,
-    path,
-) where {T <: StaticTimeSeries}
-    return _write_time_series_attributes_common!(storage, ts, path)
-end
-
-function _write_time_series_attributes!(
-    storage::Hdf5TimeSeriesStorage,
     ts::T,
     path,
-) where {T <: Forecast}
-    _write_time_series_attributes_common!(storage, ts, path)
-    interval = get_interval(ts)
-    HDF5.attributes(path)["interval"] = time_period_conversion(interval).value
-    return
-end
-
-function _write_time_series_attributes_common!(storage::Hdf5TimeSeriesStorage, ts, path)
-    initial_timestamp = Dates.datetime2epochms(get_initial_timestamp(ts))
-    resolution = get_resolution(ts)
+) where {T <: TimeSeriesData}
     data_type = get_data_type(ts)
     HDF5.attributes(path)["module"] = string(parentmodule(typeof(ts)))
     HDF5.attributes(path)["type"] = string(nameof(typeof(ts)))
-    HDF5.attributes(path)["initial_timestamp"] = initial_timestamp
-    HDF5.attributes(path)["resolution"] = time_period_conversion(resolution).value
     HDF5.attributes(path)["data_type"] = data_type
     return
 end
 
-function _read_time_series_attributes(
-    storage::Hdf5TimeSeriesStorage,
-    path,
-    rows,
-    ::Type{T},
-) where {T <: StaticTimeSeries}
-    return _read_time_series_attributes_common(storage, path, rows)
-end
-
-function _read_time_series_attributes(
-    storage::Hdf5TimeSeriesStorage,
-    path,
-    rows,
-    ::Type{T},
-) where {T <: Forecast}
-    data = _read_time_series_attributes_common(storage, path, rows)
-    data["interval"] = Dates.Millisecond(HDF5.read(HDF5.attributes(path)["interval"]))
-    return data
+function _read_time_series_attributes(path)
+    return Dict(
+        "type" => _read_time_series_type(path),
```

Comment:
We should try to remove the need to store these in HDF5 attributes. Perhaps they can be in the metadata instead. Likely out of scope for this PR.